### PR TITLE
Add Initial config for backend

### PR DESCRIPTION
**Pull request by aiorgulescu**
- Create client and server folder to modularize UI and server
- Updated README to include instructions on how to get server up and running

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,14 +13,26 @@
...
+2 - Run `npm i`
 
-2 - Run `npm start`
+3 - Run `npm run dev`
 
-3 - Go to `http://localhost:3000/`
+4 - Terminal should say `Server running on port <Number>`
 
-4 - Explore OuterWhorld!
+## It's recommended you have both UI and Server up and Running at the same time!
```
Comment (hieronymus-jodi) [inline README.md:38]: Does it matter which one's running first?

**Comment by hieronymus-jodi** [comment]
Looks good to me!! Thanks :v)

File: README.md (hunk 1 above)
Comment (hieronymus-jodi) [reply README.md:38]: I see it doesn't matter now, but do you anticipate it will matter in the future?

**Review by marden-erin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (aiorgulescu) [reply README.md:38]: mmm good question. In the future when we deploy our code I think it'll be up and running constantly so it shouldn't matter. 

If the UI is up and running but the server isn't, then the API requests will fail. So I think "technically" it's better to have the server up and running before the client. But you should have the two running before you begin coding anways, so I don't think it matters that much in the end.

**Pull request merged**